### PR TITLE
[8.0][crm_sale_marketing] - Various fixes

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
This PR addresses various fixes:

* Incorrect model used in crm.make.sale
* Security is not required in crm.make.sale
* Tests were not considering relevant security

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
@darshan-serpent @lreficent @aheficent can you review?

**Comment by serpentcs-dev1** [comment]
LGTM :+1:

File: crm_sale_marketing/README.rst [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
+==================
 
-This module extends the functionality of sale_crm module which copies the marketing data  
-from lead to sale and allows one to keep track of sales generated from marketing source.
+This module extends the functionality of sale_crm module and copies the
+marketing data from lead/opportunity to the quotation, and allows one to keep
+track of sales generated from marketing source.
 
 Installation
 ============
```
Comment (serpentcs-dev1) [inline crm_sale_marketing/README.rst]: If Installation and Configuration are empty we can remove that from README.

Otherwise the code LGTM

**Review by LoisRForgeFlow — APPROVED** [review]
Tested on runbot + code review :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM :+1:

**Pull request merged**